tools/toolchains/arm.py

```diff
@@ -31,12 +31,18 @@
 
 ARMC5_MIGRATION_WARNING = (
     "Warning: We noticed that you are using Arm Compiler 5. "
-    "We are deprecating the use of Arm Compiler 5 soon. "
+    "We are deprecating the use of Arm Compiler 5. "
     "Please upgrade your environment to Arm Compiler 6 "
     "which is free to use with Mbed OS. For more information, "
     "please visit https://os.mbed.com/docs/mbed-os/latest/tools/index.html"
 )
 
+UARM_TOOLCHAIN_WARNING = (
+    "Warning: We noticed that you are using uARM Toolchain. "
+    "We are deprecating the uARM Toolchain. "
+    "For more information on how to use the ARM toolchain with small C libraries,"
```

Comment:
Don't you need a space here like every other lines?
```suggestion
    "For more information on how to use the ARM toolchain with small C libraries, "
```

Reply:
without space it prints like that ```For more information on how to use the ARM toolchain with small C libraries,please visit ...```